<?xml version="1.0"?>

<launch>
  <node pkg="crazyflie" type="crazyflie_server.py" name="crazyflie_server" output="screen">
  </node>
</launch>
